### ISSUE TITLE
Image Block: missing title attribute (regression?)

### ISSUE THREAD

**Issue by mrfoxtalbot** [WordPress/gutenberg#32668]
There is no way to add a title attribute to an image block, the way the "alt text" _can_ be added.

This PR https://github.com/WordPress/gutenberg/pull/11070 was created in late 2019 to add a title attribute to images, but it looks like either it was never merged or it was lost later on due to a regression (I could not find it mentioned anywhere later).

This was noticed by @carinapilar here https://github.com/Automattic/wp-calypso/issues/53547 ona a WordPRess.com site but I was able to reproduce in both on a self-hosted site (WP 5.7.2) with and without the plugin (10.8).

Related: https://github.com/WordPress/gutenberg/issues/14781

**Comment by carolinan** [comment]
The title attribute option is in the Advanced section:

![An input field for the title attribute is under the Advanced settings section in the block settings sidebar](https://user-images.githubusercontent.com/7422055/134331231-a25df0ab-9e97-4c79-9643-554bc281ba7f.png)

I believe that issue can be closed?

**Comment by mrfoxtalbot** [comment]
I did not know that 😬. I'm closing this, thank you @carolinan !
